### PR TITLE
feat: upgrade airbyte to 0.50.25

### PR DESCRIPTION
**Pull request by kwigley**
## Checklist

- [ ]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [ ]  AWS
    - [ ]  GCP

### CODE REVIEW

File: airbyte/helm/airbyte/Chart.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 dependencies:
 - name: airbyte
-  repository: file://../../../../airbyte/charts/airbyte
-  version: 0.43.22
+  repository: https://airbytehq.github.io/helm-charts
```
Comment (kwigley) [inline airbyte/helm/airbyte/Chart.lock:3]: @michaeljguarino if there are upstream changes in your fork that should be included, I'll just need a hand to incorporate those! Thanks

Comment (michaeljguarino) [reply airbyte/helm/airbyte/Chart.lock:3]: yea my fork has some meaningful changes from last i remember

Comment (michaeljguarino) [reply airbyte/helm/airbyte/Chart.lock:3]: could you revert that for now as we can't merge with it as a dependency (you actually shouldn't need to have my fork to perform this upgrade as the chart is in the repo and should be safe to apply w/

Comment (michaeljguarino) [reply airbyte/helm/airbyte/Chart.lock]: yep that's it

**Review by michaeljguarino — COMMENTED** [review]
I'm seeing an issue where airbyte-webapp can't bind port 80 and so cannot run (likely the base image that they're using no longer runs as root).  I don't see an easy fix there so we might need to pause until there's a way to solve it.

**Comment by kwigley** [comment]
Ah ya, I see. I tested with their most recent helm chart and everything I seemed to be working properly. Sounds good, I'll close for now.

**Pull request closed without merging**